packaging/docker/release/Dockerfile

```diff
@@ -20,27 +20,28 @@
 FROM ubuntu:18.04
 
 RUN apt-get update && \
-	apt-get install -y curl>=7.58.0-2ubuntu3.6 \
-		dnsutils>=1:9.11.3+dfsg-1ubuntu1.7 \
-		lsof>=4.89+dfsg-0.1 \
-		tcptraceroute>=1.5beta7+debian-4build1 \
-		telnet>=0.17-41 \
-		netcat>=1.10-41.1 \
-		strace>=4.21-1ubuntu1 \
-		tcpdump>=4.9.3-0ubuntu0.18.04.1 \
-		less>=487-0.1 \
-		vim>=2:8.0.1453-1ubuntu1.4 \
-		net-tools>=1.60+git20161116.90da8a0-1ubuntu1 \
-		jq>=1.5+dfsg-2 && \
+	apt-get install -y curl \
+		dnsutils \
+		lsof \
+		tcptraceroute \
+		telnet \
+		netcat \
+		strace \
+		tcpdump \
+		less \
+		vim \
+		net-tools \
+		jq && \
```

Comment:
Any reason to remove the version pinning?

Reply:
apt doesn't support version pinning through the cli nor it supports ">=" as a version specifier. I have explained the rationale in the Description. Currently the root directory of the images look like this:
```bash
$ ls -alh
total 40K
drwxr-xr-x.   1 root root  33 Sep 29 16:53  .
drwxr-xr-x.   1 root root  33 Sep 29 16:53  ..
-rw-r--r--.   1 root root   0 Jun 25 04:12 '=0.17-41'
-rw-r--r--.   1 root root 39K Jun 25 04:13 '=1.1.1-1ubuntu2.1~18.04.9'
-rw-r--r--.   1 root root   0 Jun 25 04:12 '=1.10-41.1'
-rw-r--r--.   1 root root   0 Jun 25 04:12 '=1.5+dfsg-2'
-rw-r--r--.   1 root root   0 Jun 25 04:12 '=1.5beta7+debian-4build1'
-rw-r--r--.   1 root root   0 Jun 25 04:12 '=1.60+git20161116.90da8a0-1ubuntu1'
-rw-r--r--.   1 root root   0 Jun 25 04:12 '=1:9.11.3+dfsg-1ubuntu1.7'
-rw-r--r--.   1 root root   0 Jun 25 04:12 '=2:8.0.1453-1ubuntu1.4'
-rw-r--r--.   1 root root   0 Jun 25 04:12 '=4.21-1ubuntu1'
-rw-r--r--.   1 root root   0 Jun 25 04:12 '=4.89+dfsg-0.1'
-rw-r--r--.   1 root root   0 Jun 25 04:12 '=4.9.3-0ubuntu0.18.04.1'
-rw-r--r--.   1 root root   0 Jun 25 04:12 '=487-0.1'
-rw-r--r--.   1 root root   0 Jun 25 04:12 '=7.58.0-2ubuntu3.6'
<snip>
```

Reply:
I was not arguing that the `>=` was wrong, I was rather asking why we don't want to pin the version.

Reply:
I never said we don't want to pin but:
1. We are not pinning currently at all (except for `tini`)
2. This change just merely cleans up the `apt` commands involved.
3. While a massive fan of pinning, since `fdbserver`, `fdbcli` and the relevant binaries do not depend on any of those utilities, I do not see a value on pinning the development tools. I'm content with pulling in the latest updated binaries from ubuntu's upstreams.
4. If we do see a value in pinning the debugging tools versions, I'll be happy to create the relevant preference files.